docs/apache-airflow-providers-amazon/executors/ecs-executor.rst

```diff
@@ -234,6 +213,41 @@ your host machine to the container's ``/home/airflow/.aws`` directory.
 Keep in mind the Docker build context when copying the ``.aws`` folder
 to the container.
 
+Base Image
+~~~~~~~~~~
+
+The Docker image created above is built upon the ``apache/airflow:latest`` image. See
+`here <https://hub.docker.com/r/apache/airflow>`__ for more information
+about the image.
+
+Important note: The Airflow and python versions in this image must align
+with the Airflow and python versions on the host/container which is
+running the Airflow scheduler process (which in turn runs the executor).
+The Airflow version of the image can be verified by running the
+container locally with the following command:
+
+.. code-block:: bash
+
+   docker run my-airflow-image version
```

Comment:
Moved this section below the first image build commands so that they actually have an image to run with. I also updated this line (and the one below) to have the image name we use by default in the build commands.